lib/electoral_history.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class ElectoralHistory < Scraped::HTML
+  def to_h
```

Comment:
Overriding the `to_h` method of a `Scraped::HTML` subclass seems wrong. Why do you need to do this?